data/tutorials/getting-started/gs_07_operators.md

```diff
@@ -0,0 +1,196 @@
+  * `%` `<` or
+  * `!` `.` `:` `?` `~`
+
+This is defined in the [Prefix and infix symbols](https://v2.ocaml.org/releases/5.1/htmlman/lex.html#sss:lex-ops-symbols) section of The OCaml Manual.
+
+Tips:
+  * Don't define wide scope operators. Restrict their scope to module or function.
+  * Don't use many of them
+  * Before defining a custom binary operator, check if the symbol is not already used. This can be done in two ways
+    - By surrounding the candidate symbol with parentheses in UTop and see if it responds with a type or `Unbound value`
+    - Use [Sherlocode](https://sherlocode.com/) to check if it is already used in some OCaml project
+  * Avoid shadowing existing operators
+
+## Operator Associativity and Precedence
+
+Let's illustrate what operator associativity is using an example. Consider this function, it concatenates its string arguments, surrounded by `|` characters and separated by a `_` character.
+```ocaml
+# let par s1 s2 = "|" ^ s1 ^ "_" ^ s2 ^ "|";;
+val par : string -> string -> string = <fun>
+
+# par "hello" "world";;
+- : string = "|hello_world|"
+```
+
+Let's turn `par` into two different operators.
+```ocaml
+# let ( @^ ) = par;;
+val ( @^ ) : string -> string -> string = <fun>
+
+# let ( &^ ) = par;;
+val ( &^ ) : string -> string -> string = <fun>
+```
+
+At first sight, operators `@^` and `&^` are the same. However, the OCaml parser allows forming expression using several operators without parentheses.
+```ocaml
+# "foo" @^ "bar" @^ "bus";;
+- : string = "|foo_|bar_bus||"
+
+# "foo" &^ "bar" &^ "bus";;
+- : string = "||foo_bar|_bus|"
+```
+
+Although both expressions are calling the same function (`par`), they are evaluated in different order.
+1. Expression `"foo" @^ "bar" @^ "bus"` is evaluated as if it was `"foo" @^ ("bar" @^ "bus")`. Parentheses are added at the right, therefore `@^` _associates to the right_
+1. Expression `"foo" &^ "bar" &^ "bus"` is evaluated as if if was `"(foo" @^ "bar") @^ "bus"`. Parentheses are added at the left, therefore `@^` _associates to the left_
+
+Operator _precedence_ rules how expression combining several operators without parentheses are interpreted. For instance, using the same operators, here is how expressions using both are evaluated.
+```ocaml
+# "foo" &^ "bar" @^ "bus";;
+- : string = "(foo, (bar, bus))"
+
+# "foo" @^ "bar" &^ "bus";;
+- : string = "((foo, bar), bus)"
+```
+
+In both cases, values are passed to `@^` before `&^`. Therefore, it is said that `@^` has higher _precedence_ over `&^`. Rules for operator priorities are detailed in the [Expressions](https://v2.ocaml.org/manual/expr.html#ss%3Aprecedence-and-associativity) section of the OCaml Manual. They can be summarized the following way. The first character of an operator dictates its associativity and priority. Here are the groups operators of first character, each group has same associativity and precedence, groups are sorted in increasing precedence order.
```

Comment:
```suggestion
In both cases, values are passed to `@^` before `&^`. Therefore, it is said that `@^` has higher _precedence_ over `&^`. Rules for operator priorities are detailed in the [Expressions](https://v2.ocaml.org/manual/expr.html#ss%3Aprecedence-and-associativity) section of the OCaml Manual. In short, an operator's first character dictates its associativity and priority. Here are the groups operators of first character. Each group has the same associativity and precedence, and groups are sorted in increasing precedence.
```

I'm stuck on this sentence: "Here are the groups operators of first character. "
Here are the groups' operators' first character.
Here are the group's operators' first character. 
Here are the groups, showing the operators' first character.
... or something else?

If you can help clarify the meaning, I will revise the sentence. Thank you!

Reply:
This one sentence still needs some clarity.